src/mero_conn.erl

```diff
@@ -69,6 +83,14 @@ delete(Name, Key, Timeout) ->
     pool_execute(PoolName, delete, [Key, TimeLimit], TimeLimit).
 
 
+mdelete(Name, Keys, Timeout) ->
+    TimeLimit = mero_conf:add_now(Timeout),
+    KeysGroupedByShards = mero_cluster:group_by_shards(Name, Keys),
+    async_by_shard(Name, KeysGroupedByShards, TimeLimit,
+                   [async_delete, async_delete_error, async_blank_response,
+                    async_delete_response_error]),
+    ok.
```

Comment:
We should give a return that gives a hint on wether there was an error or not sending the request and not just return "ok"